src/checkovRunner.ts

```diff
@@ -58,7 +58,7 @@ export const runCheckovScan = (logger: Logger, checkovInstallation: CheckovInsta
         let stdout = '';
 	
         ckv.stdout.on('data', data => {
-            if (data.toString().startsWith('{') || stdout) {           
+            if (data.toString().startsWith('{') || data.toString().startsWith('[') || stdout) {           
```

Comment:
Why do you need all this? 🤔

Reply:
Because sometimes Checkov prints log before the output of the checks

Reply:
Got it 👌